lang/cs/Org.Apache.REEF.Wake/Remote/Proto/WakeRemoteProtosGen.cs

```diff
@@ -29,27 +29,27 @@
 // Generated from: src/main/proto/RemoteProtocol.proto
 namespace Org.Apache.REEF.Wake.Remote.Proto
 {
-  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name=@"WakeMessagePBuf")]
+  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name = @"WakeMessagePBuf")]
```

Comment:
What I suspected is StyleCop's detecting feature for auto-generated codes. 
For Avro, StyleCop is know to recognize `// <auto-generated />` at the top of file. 
Apache License might make StyleCop confused. Anyway, I didn't exclude for Proto files, it does not block builds.